app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/tests/activities/SplashActivityTest.java

```diff
@@ -1,87 +0,0 @@
-package org.kiwix.kiwixmobile.tests.activities;
```

Comment:
Is there a reason this is deleted? Can you comment it out instead? Same with a lot of the tests that have been deleted. Generally deleting tests is a bad idea.

Reply:
This has been renamed to **org.kiwix.kiwixmobile.main.MainActivityTest**.
I have not deleted any test. Instead of commenting out the tests I have fixed them.

- Just **org/kiwix/kiwixmobile/tests/activities/MainActivityTest.java** has been renamed to **org/kiwix/kiwixmobile/splash/SplashActivityTest.java** as we only needed to test the launch of SplashActivity. 
- And **org/kiwix/kiwixmobile/tests/activities/SplashActivityTest.java** has been renamed to **java/org/kiwix/kiwixmobile/main/MainActivityTest.java** as in this file we are testing **MainActivity** not **SplashActivity**.

Doing so will follow the same package structure that of the source packages. So we don't have to make the scope of many classes public unnecessarily.

I have only deleted a single file in tests which is **org/kiwix/kiwixmobile/tests/KiwixMobileActivityTest.java** and that was empty, containing absolutely no tests.

Reply:
Ah my bad. thanks for clarifying this.